util/util.go

```diff
@@ -634,3 +635,16 @@ func MergeMap(maps ...map[string]string) map[string]string {
 	}
 	return m
 }
+
+// DefaultKubeadmConfigSpec defaults a KubeadmConfigSpec.
+func DefaultKubeadmConfigSpec(r *bootstrapv1.KubeadmConfigSpec) {
+	if r.Format == "" {
+		r.Format = bootstrapv1.CloudConfig
+	}
+	if r.InitConfiguration != nil && r.InitConfiguration.NodeRegistration.ImagePullPolicy == "" {
+		r.InitConfiguration.NodeRegistration.ImagePullPolicy = "IfNotPresent"
+	}
+	if r.JoinConfiguration != nil && r.JoinConfiguration.NodeRegistration.ImagePullPolicy == "" {
+		r.JoinConfiguration.NodeRegistration.ImagePullPolicy = "IfNotPresent"
+	}
+}
```

Comment:
I think this just wants to be in the defaulting webhook itself, then use the webhook in other packages that need this defautling